docs-kits/kits/behaviour-twin-kit/use-cases/rul/adoption-view/architecture.md

```diff
@@ -0,0 +1,83 @@
+﻿---
+id: architecture
+title: Architecture
+description: Behaviour Twin KIT
+---
+
+<div style={{display:'block'}}>
+  <div style={{display:'inline-block', verticalAlign:'top'}}>
+
+![Behaviour Twin KIT -- Remaining useful Life banner](@site/static/img/kit-icons/behaviour-twin-rul-kit-icon-mini.svg)
+
+  </div>
+  <div style={{display:'inline-block', fontSize:17, color:'rgb(255,166,1)', marginLeft:7, verticalAlign:'top', paddingTop:6}}>
+Behaviour Twin KIT -- Remaining useful Life
+  </div>
+</div>
+
+## BASIC ARCHITECTURE
+
+### OVERVIEW
+
+An external consumer wants to know the remaining useful life of its component (e.g. a vehicle). It uses a RuL Consuming App (e.g. with a web interface) for this. At the data provider (likely an OEM), usage data is collected continuously with the consent and on behalf of the consumer. The usage history and all dependent components that can provide RuL values are known on the data provider's side. The data provider itself doesn't even have to know which components can provide RuL services/values. Its registed suppliers provide that information in their catalogs which are synchronized periodically. The data provider only must bind the usage data to the knowledge graph.
+
+This approach is not limited only to OEMs and vehicles. Any system that provides usage data can utilize this approach, like tools/machines, production lines, building facilities, vending machines, computers, roads and so on.
+
+### ROLES
+
+Each participant in the Remaining useful Life (RuL) Behaviour Twin use case applies to one or more of the following roles:
+
+- **RuL consumer** (result requester)
+- **skill provider** (Consumer, provider of the use case logic)
+- **data provider** (OEM, provider of usage data)
+- **RuL service provider** (a component supplier, e.g. a gearbox supplier)
+
+One of the special cases would be that the data provider is also the RuL consumer and/or RuL service provider and/or skill provider.
+
+## DETAILED ARCHITECTURE
+
+### BUSINESS PROCESS
+
+*For better readability will be on this page from now on assuming that the **data provider** is always an **OEM**.*
+
+The first addressee of a RuL skill must be the OEM since it has access to its suppliers who can provide RuL values.
+
+![business-process](assets/business-process.drawio.svg)
+
+0. **sync federated catalog:** <br/> The federated catalogs are synchronized periodically. As a result, the OEM can resolve RuL prognosis function assets that are located at the supplier.
```

Comment:
Minor remark: Markdown notation / linebreaks should be preferred to an HTML-tag

Reply:
The desired result (line break within a list element) is not possible without HTML as I know. If there is a better solution, please let me know.